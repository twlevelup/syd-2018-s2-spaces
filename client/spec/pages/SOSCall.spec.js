const SOSCall = require('../../src/js/pages/SOSCall');

describe('SOSCall', () => {
  describe('#render', () => {
    it('should render SOS Call', () => {
      const page = new SOSCall();
      expect(page.render()).toMatchSnapshot()
    }); //do not touch formatting haha
  });
  describe('#topButtonEvent', () => {
    it('goes to Home page', () => {
      const props = {
        navigate: () => { },
      };

      const page = new SOSCall(props);
      spyOn(page, 'navigate');

      page.topButtonEvent();
      expect(page.navigate).toHaveBeenCalledWith('/');

    });
  });
});
