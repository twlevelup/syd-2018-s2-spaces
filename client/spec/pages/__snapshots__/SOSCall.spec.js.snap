// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SOSCall #render should render SOS Call 1`] = `
"<div class=\\"container\\">
    <div>
        <img style=\\"width: 30px; height: 30px; text-align:center\\" src=\\"../images/homeicon.png\\">
        <div>Cancel</div>
    </div>

    <div>
        <img src='../images/Phone.png' style=\\"text-align:center; height: 25%; width: 25%;\\">
        <div>Calling Police...</div>
    </div>
    <div>
        <br>
    </div>
</div>
"
`;
